src/chocolatey/infrastructure.app/commands/ChocolateyApiKeyCommand.cs

```diff
@@ -53,28 +50,57 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                 .Add("k=|key=|apikey=|api-key=",
                      "ApiKey - The API key for the source. This is the authentication that identifies you and allows you to push to a source. With some sources this is either a key or it could be a user name and password specified as 'user:password'.",
                      option => configuration.ApiKeyCommand.Key = option.remove_surrounding_quotes())
-                .Add("rem|remove",
-                    "Removes an API key from Chocolatey (DEPRECATED)",
-                    option => configuration.ApiKeyCommand.Remove = true)
                 ;
         }
 
         public virtual void handle_additional_argument_parsing(IList<string> unparsedArguments, ChocolateyConfiguration configuration)
         {
             configuration.Input = string.Join(" ", unparsedArguments);
+
+            if (unparsedArguments.Count > 1)
+            {
+                throw new ApplicationException("A single sources command must be listed. Please see the help menu for those commands");

```

Comment:
Is this a copy/pasta? This should be talking about the apikey command, right?

Reply:
Yes it is, I thought I got all these, but I guess I missed one. Will fix tomorrow, thanks!

Reply:
Sorted :)